src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -143,24 +154,47 @@ internal async Task InitializeJobhostLanguageWorkerChannelAsync(int attemptCount
             }
         }
 
+        internal async Task InitializeJobhostLanguageWorkerChannelAsync(int attemptCount, IEnumerable<string> languages)
```

Comment:
Should the method name have _channels_ instead of _channel_ as it is dealing with multiple languages and channels?

Reply:
I think this is because its an overload of `InitializeJobhostLanguageWorkerChannelAsync`